Tasks/KubernetesManifestV0/Tests/TestSetup.ts

```diff
@@ -13,14 +13,14 @@ const definitionId = '123';
 const teamFoundationCollectionUri = 'https://abc.visualstudio.com/';
 const jobName = 'jobName';
```

Comment:
We should have a test case where user does not provide the namespace and the endpoint does not have the namespace and it fails.

We should have another test case where namespace is used from the endpoint

Reply:
I have added tests for the latter; The majority of the tests would use the namespace from the endpoint; except for 2 tests, where I explicitly added namespace as an input.